src/MyInfoConnector/MyInfoConnector.cs

```diff
@@ -0,0 +1,293 @@
+        /// The access token is then uses to invoke the person API to get the Person data.
+        /// </summary>
+        public string GetPersonJson(
+            string redirectUri
+            ,string authCode
+            ,string state = null
+            ,string transactionId = null)
+        {
+            string result;
+            string jsonResponse = null;
+
+            string token = GetAccessToken(redirectUri, authCode, state);
+
+            if (string.IsNullOrEmpty(token))
+            {
+                // Not authorised or something - either way cannot continue
+                return null;
+            }
+
+            var jObject = JObject.Parse(MyInfoSecurityHelper.DecodeToken(token).ToString());
+            string uinfin = (string) jObject.SelectToken("sub");
+
+            // GET PERSON
+            result = GetPersonJsonWorker(uinfin, "Bearer " + token,  transactionId);
+
+            if (_config.Environment == ApplicationConstant.SANDBOX)
+            {
+                jsonResponse = result;
+            }
+            else
+            {
+                try
+                {
+                    jsonResponse = DecodeTokenToPerson(result);
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"{nameof(GetPersonJson)} failed to decode the encrypted result: {ex.Message}" );
+                }
+            }
+
+            return jsonResponse;
+        }
+
+
+
+        /// <summary>
+        /// This API is invoked by your application server to obtain an "access
+        /// token", which can be used to call the Person API for the actual data.
+        /// Your application needs to provide a valid "authorisation code" from the
+        /// authorise API in exchange for the "access token".
+        /// </summary>
+        protected string GetAccessToken(string redirectUri, string authCode, string state)
+        {
+            string baseParams;
+            string accessToken;
+            string baseString = string.Empty;
+            string authHeader = string.Empty;
+
+            try
+            {
+                var nonce = MyInfoSecurityHelper.GetRandomInteger();
+                long timestamp = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
+
+                string signature = null;
+
+                // A) Forming the Signature Base String
+                baseParams = $"{ApplicationConstant.APP_ID}={_config.ClientAppId}&{ApplicationConstant.CLIENT_ID}={_config.ClientAppId}&{ApplicationConstant.CLIENT_SECRET}={_config.ClientAppPassword}&{ApplicationConstant.CODE}={authCode}&{ApplicationConstant.GRANT_TYPE}={ApplicationConstant.AUTHORIZATION_CODE}&{ApplicationConstant.NONCE}={nonce}&{ApplicationConstant.REDIRECT_URI}={redirectUri}&{ApplicationConstant.SIGNATURE_METHOD}={ApplicationConstant.RS256}&{ApplicationConstant.STATE}={state}&{ApplicationConstant.TIMESTAMP}={timestamp}"; ;
```

Comment:
Extra `;` at the end.
```suggestion
                baseParams = $"{ApplicationConstant.APP_ID}={_config.ClientAppId}&{ApplicationConstant.CLIENT_ID}={_config.ClientAppId}&{ApplicationConstant.CLIENT_SECRET}={_config.ClientAppPassword}&{ApplicationConstant.CODE}={authCode}&{ApplicationConstant.GRANT_TYPE}={ApplicationConstant.AUTHORIZATION_CODE}&{ApplicationConstant.NONCE}={nonce}&{ApplicationConstant.REDIRECT_URI}={redirectUri}&{ApplicationConstant.SIGNATURE_METHOD}={ApplicationConstant.RS256}&{ApplicationConstant.STATE}={state}&{ApplicationConstant.TIMESTAMP}={timestamp}";
```